src/components/studies/scheduler/utility.ts

```diff
@@ -1,6 +1,22 @@
 import moment from 'moment'
 import {StringDictionary} from '../../../types/types'
-import {Schedule} from '../../../types/scheduling'
+import {Schedule, HDWMEnum} from '../../../types/scheduling'
+
+export function getTimeExpiredAfter(durationString: string) {
```

Comment:
I think this function should be renamed to be more descriptive of what it's actually doing